skore/src/skore/project/project.py

```diff
@@ -117,47 +116,27 @@ def put(self, key: Union[str, dict[str, Any]], value: Optional[Any] = None):
         value : Any, optional
             The value to associate with ``key`` in the Project.
             If ``key`` is a dict, this argument is ignored.
-
-        Raises
```

Comment:
Can you replace the Raises docstring by the new possible exception types?

Reply:
of course, fixed